docs/version0.49.md

```diff
@@ -37,6 +38,12 @@ macOS 12 is out of support and is removed from the list of supported platforms.
 
 To learn more about support for OpenJ9 releases, including OpenJDK levels and platform support, see [Supported environments](openj9_support.md).
 
+### Change to the shared classes cache generation number
+
+A new flag `J9AccClassIsShared` is added to ROMClasses to indicate whether a ROMClass was loaded from a shared classes cache or from a VM. On all platforms, the format of ROMClasses that are stored in the shared classes cache is changed. As a result of this change, the shared classes cache generation number is incremented. This increment causes the VM to create a new shared classes cache, rather than re-creating or reusing an existing cache.
```

Comment:
I suggest we put the more relevant details first (starting from "the shared classes cache generation number is incremented" and what to do), with the explanation of why it changed afterwards. Most users won't care why it changed.

Reply:
@Sreekala-Gopakumar you can also include information on how to destroy the outdated shared class caches similar to what was included in https://github.com/eclipse-openj9/openj9-docs/pull/347

Reply:
@theresa-m - You mean the link to the `-Xshareclasses` topic? I have added the line "For more information, see [Housekeeping](shrc.md#housekeeping) and [`-Xshareclasses`](xshareclasses.md)." Is that ok?

Reply:
Yes, sounds good.